run.go

```diff
@@ -781,6 +781,7 @@ func setupNamespaces(g *generate.Generator, namespaceOptions NamespaceOptions, i
 				return false, nil, false, errors.Wrapf(err, "error adding new %q namespace for run", string(specs.NetworkNamespace))
 			}
 			configureNetwork = (policy != NetworkDisabled)
+			g.Spec().Linux.Sysctl = util.DefaultNetworkSysctl
```

Comment:
I think you'd need to duplicate this at line 732 to get the same effect.

Reply:
This should probably be call to `g.AddLinuxSysctl()`, since we try not to mix using the Generator object's methods with direct modifications to a Spec structure.

Reply:
I looked for that but did not see it.